enterprise/cypher/acceptance-spec-suite/src/test/resources/cypher/features/DurationBetweenAcceptance.feature

```diff
@@ -359,25 +517,109 @@ Feature: DurationBetweenAcceptance
+              duration.seconds(localdatetime("2015-07-21T21:40:32.142"), date("2015-06-24")),
               duration.seconds(localdatetime("2015-07-21T21:40:32.142"), localdatetime("2016-07-21T21:45:22.142")),
               duration.seconds(localdatetime("2015-07-21T21:40:32.142"), datetime("2015-07-21T21:40:32.142+0100")),
-              //duration.seconds(localdatetime("2015-07-21T21:40:32.142"), localtime("16:30")),
-              //duration.seconds(localdatetime("2015-07-21T21:40:32.142"), time("16:30+0100")),
+              duration.seconds(localdatetime("2015-07-21T21:40:32.142"), localtime("16:30")),
+              duration.seconds(localdatetime("2015-07-21T21:40:32.142"), time("16:30+0100")),
 
-              //duration.seconds(datetime("2014-07-21T21:40:36.143+0200"), date("2015-06-24")),
+              duration.seconds(datetime("2014-07-21T21:40:36.143+0200"), date("2015-06-24")),
               duration.seconds(datetime("2014-07-21T21:40:36.143+0200"), localdatetime("2016-07-21T21:45:22.142")),
-              duration.seconds(datetime("2014-07-21T21:40:36.143+0200"), datetime("2015-07-21T21:40:32.142+0100"))
-              //duration.seconds(datetime("2014-07-21T21:40:36.143+0200"), localtime("16:30")),
-              //duration.seconds(datetime("2014-07-21T21:40:36.143+0200"), time("16:30+0100"))
+              duration.seconds(datetime("2014-07-21T21:40:36.143+0200"), datetime("2015-07-21T21:40:32.142+0100")),
+              duration.seconds(datetime("2014-07-21T21:40:36.143+0200"), localtime("16:30")),
+              duration.seconds(datetime("2014-07-21T21:40:36.143+0200"), time("16:30+0100"))
               ] as d
       RETURN d
       """
     Then the result should be, in order:
       | d |
+      | 'PT269112H' |
+      | 'PT278565H45M22.142S' |
+      | 'PT269781H40M32.142S' |
+      | 'PT16H30M' |
+      | 'PT16H30M' |
+      | 'PT-14H-30M' |
+      | 'PT7H15M22.142S' |
+      | 'PT7H10M32.142S' |
+      | 'PT2H' |
+      | 'PT2H' |
+      | 'PT-14H-30M' |
+      | 'PT7H15M22.142S' |
+      | 'PT6H10M32.142S' |
+      | 'PT2H' |
+      | 'PT1H' |
+      | 'PT-669H-40M-32.142S' |
+      | 'PT8784H4M50S' |
+      | 'PT0S' |
+      | 'PT-5H-10M-32.142S' |
+      | 'PT-5H-10M-32.142S' |
+      | 'PT8090H19M23.857S' |
+      | 'PT17544H4M45.999S' |
+      | 'PT8760H59M55.999S' |
+      | 'PT-5H-10M-36.143S' |
+      | 'PT-4H-10M-36.143S' |
+    And no side effects
+
+  Scenario: Should compute duration between if they differ only by a fraction of a second and the first comes after the second.
+    Given an empty graph
+    When executing query:
+      """
+      RETURN duration.seconds(localdatetime("2014-07-21T21:40:36.143"), localdatetime("2014-07-21T21:40:36.142")) as d
+      """
+    Then the result should be, in order:
+      | d |
+      | 'PT-0.001S' |
+    And no side effects
+
+  Scenario: Should handle durations at daylight saving time day
```

Comment:
I was expecting there to be some tested entries `Europe/Stockholm` in summer as well to really drive home the point.

Reply:
2017-10-29 00:00 is summer :) or at least summer time... This is the day when we changed the clock. Or did you have some specific test in mind?

Reply:
Ahaa, now I see how it works. Nevermind!